app/core/Router.coffee

```diff
@@ -95,8 +95,6 @@ module.exports = class CocoRouter extends Backbone.Router
     'careers': => window.location.href = 'https://jobs.lever.co/codecombat'
     'Careers': => window.location.href = 'https://jobs.lever.co/codecombat'
 
-    'cinematic': go('CinematicView')
```

Comment:
Will this route also be served from vuerouter?

Reply:
Thanks for picking this up. I've completely removed this view while we figure out how to create the vue routes with different base templates.

Reply:
In order to do this we'll have to implement base-flat in Vue and then include it as needed (for example include it in school admin dashboard).  Looked at this briefly and we because we have the raw jade component we should be able to do this quickly.  We can discuss more in eng sync.